scss/_patterns_search-and-filter.scss

```diff
@@ -116,10 +116,26 @@
     }
 
     .p-search-and-filter__box {
+      align-items: center;
       display: inline-flex;
       flex: 1;
+      height: 100%;
       margin: 0;
       position: relative;
+
+      // tab-selectable search submit button
+      & > .u-off-screen {
```

Comment:
Does this element have any other more specific class name, to target the button?

Adding all of this styling (specific to buttons) to the `u-off-screen` utility class feels bit hacky. If there ever is another element there added with u-off-screen it would also get those styles.

Reply:
Looks like there is no other class name. 

Quick and dirty way to do it could be to at least do `& > button.u-off-screen` to make it specific to the buttons (because it seems to be overriding the button styles).

Better way would be to add class name, like `p-search-and-filter__button` (?) that could get the styling.
This component is likely never used directly in HTML/CSS, so we could easily add the class name to its React implementation as well.